### ISSUE TITLE
Outlook Online, New Outlook: Outlook loses keyboard focus when add-in pane is pinned in Compose mode

### ISSUE THREAD

**Issue by DmitryKostochko** [OfficeDev/office-js#5936]
# Provide required information needed to triage your issue
We are developing an Outlook add-in for Compose mode. Recently we've started to receive complaints that Outlook loses keyboard focus when our pane is pinned. See the steps leading to the issue below. Please note that **all Outlook add-ins are affected**.

## Your Environment
* Platform: PC desktop, Office on the web
* Host: Outlook Online, New Outlook
* Office version number:
* Operating System: Windows 10, Windows 11
* Browser: Edge, Chrome
 
## Expected behavior
Outlook should not lose keyboard focus when a pinned add-in pane is open.

## Current behavior
Outlook loses keyboard focus.

## Steps to reproduce
1. In Outlook Online or New Outlook, create a new mail.
2. Start any Outlook add-in that works in Compose mode, for example Script Lab for Outlook.
3. Pin the add-in pane.
4. Close the email that you created in step 1.
5. Create a new mail again. As you see, the keyboard focus is not in the To field, where it should be.
6. Close the email.
7. Select any email in your Inbox, click Reply or Forward. As you see, the keyboard focus is neither in the Body field (in case of Reply) nor in the To field (in case of Forward), where it is supposed to be.

## Context
As you understand, this current behavior affects many users, we are receiving complaints.

**Comment by Swathy-Mothilal** [comment]
@DmitryKostochko Thanks for reporting the issue. We are able to repro the issue. It has been put on our backlog. We unfortunately have no timelines to share at this point.

Internal tracking id: Office: 5981324

**Comment by DmitryKostochko** [comment]
Hi @kustonaut,

I have just retested the issue on Windows 11 and can confirm that it is still reproducible. I tested the following:
- Outlook on the web, Chrome Version 152.0.7977.83 (Official Build) (64-bit)
- New Outlook (Microsoft Outlook Version: 1.2026.818.100, Client Version: 20260821009.11, WebView2 Version: 152.0.4191.62 (Stable))

In both cases I pinned the Script Lab for Outlook add-in.
